apps/marketplace/components/Brief/Overview/BriefOverviewSectionLinkList.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react'
+
+import BriefOverviewSectionLinkListItem from './BriefOverviewSectionLinkListItem'
+
+import styles from './BriefOverviewSectionLinkList.scss'
+
+export class BriefOverviewSectionLinkListComponent extends Component {
```

Comment:
this can be SFC

Reply:
Feels like this and parent could be combined?

Reply:
I've merged the `LinkList` and `LinkListItem` and changed to SFC.  Added prop types too.